packages/fxa-auth-server/lib/db.js

```diff
@@ -1156,6 +1156,30 @@ module.exports = (config, log, Token, UnblockCode = null) => {
     return this.pool.get(SAFE_URLS.securityEvents, { ipAddr, uid });
   };
 
+  SAFE_URLS.securityEventsByUid = new SafeUrl(
+    '/securityEvents/:uid',
+    'db.securityEventsByUid'
+  );
+  DB.prototype.securityEventsByUid = function(params) {
+    log.trace('DB.securityEventsByUid', {
+      params: params,
+    });
+    const { uid } = params;
+    return this.pool.get(SAFE_URLS.securityEventsByUid, { uid });
+  };
+
+  SAFE_URLS.deleteSecurityEventsByUid = new SafeUrl(
```

Comment:
On reflection, the `deleteSecurityEventsByUid` in can be improved by simply calling it `deleteSecurityEvents`. The functions in here assume you are passing either `uid` or `id` when they are called. 

However, we don't have to the change the `securityEventsByUid` since we already have `securityEvents` function.